### ISSUE TITLE
TypeOrm cause MaxListenersExceededWarning

### ISSUE THREAD

**Issue by michaelsogos** [nestjs/typeorm#636]
With version 0.2.25 of TypeOrm, when I try to save 10 records at once using Repository.Save(), nodejs print the "MaxListenersExceededWarning".

Typeorm project owners already knows about "MaxListenersExceededWarning" with PGSQL server (exactly my case).
[Typeorm Issue Here](https://github.com/typeorm/typeorm/pull/6708)

Actually i don't want to add manually the package typeorm@0.2.26, i prefer to let this library to include the right version of typeorm; so i will wait till a newer package version will be released which will include a newer package version of typeorm.

**Comment by kamilmysliwiec** [comment]
Fixed

**Comment by climmmm** [comment]
Hello,

Sorry, but we always have the warning displayed when we want to register an entity in the PostgresSQL database: 

``
(node:11170) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 error listeners added to [Client]. Use emitter.setMaxListeners() to increase the limit.
``

Code sample:  
```typescript
import { Injectable } from '@nestjs/common';
import { Repository } from 'typeorm';
import { GbfsRt } from './gbfs-rt.entity';
import { InjectRepository } from '@nestjs/typeorm';
@Injectable()
export class DbGbfsRtService {
  constructor(
    @InjectRepository(GbfsRt)
    private readonly dbGbfsRtRepository: Repository<GbfsRt>,
  ) {
  }

  async saveAll(gbfsRts: GbfsRt[]): Promise<void> {
    try {
      await this.dbGbfsRtRepository.save(gbfsRts);
    } catch (error) {
      throw error;
    }
  }
}
```


The warning has appeared since we upgraded the `@nestjs/typeorm` and `typeorm` packages :  
<img width="409" alt="Capture d’écran 2020-09-22 à 11 31 41" src="https://user-images.githubusercontent.com/53145712/93868605-f3142b00-fcca-11ea-8667-71c926345128.png">

**Comment by sclausen** [comment]
@climmmm It's fixed in the PR typeorm/typeorm#6708 and discussed in the issue typeorm/typeorm#6699, but not released yet.

**Comment by climmmm** [comment]
Thank you !
